backend/test_streaming_fix.py

```
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
```

Ruff (0.14.2)

Shebang is present but file is not executable

(EXE001)